app/controllers/casework/assigned_applications_controller.rb

```diff
@@ -66,7 +63,16 @@ def require_a_user_work_stream
       return unless current_user.work_streams.empty?
 
       set_flash(:no_work_streams_to_assign_from, success: false)
-      render :index
+      set_search_filters
```

Comment:
@hiboabd I don't think you need to set search filters now this action has changed from using "render" to "redirect_to"